### DISCUSSION TITLE
Need Help with MotionPlus

### DISCUSSION THREAD

**Discussion by AKalakota23** [Motion-Project/motionplus#183] [Q&A]
Hello, I am encountering some issues when trying to run motionplus. I am not able to open the webserver and the recorded video appears to be really laggy. Here are some errors I am greeted with in the logs:
![IMG_9350](https://github.com/user-attachments/assets/68ee5a99-da01-441e-a1ad-9c18e6f14ed8)

For some cotext of my setup, I am using a raspberry pi zero 2 w with debian Bookworm (32 bit) os installed with a raspberry pi camera module v2. I recently installed motionplus using these commands:
To download the .deb file
`wget https://github.com/Motion-Project/motionplus/releases/download/<version>/motionplus-<version>-armhf.deb`

To install the .deb package
`sudo dpkg -i motionplus-<version>-armhf.deb`
`sudo apt --fix-broken install  # To resolve any dependencies`

I ran motionplus with `sudo motionplus` and those messages popped up in the logs

Here is my config file I accessed by running `sudo nano /etc/motionplus/motionplus.conf`:

; This config file was generated by motionplus 0.2.1
; Documentation: /usr/share/doc/motionplus/motionplus_guide.html
;
; This file contains only the basic configuration options to get a
; system working. There are many more options available. Please
; consult the documentation for the complete list of all options.
;

;*************************************************
;***** System
;*************************************************
daemon off
setup_mode off
;log_file 
log_level 6
log_type ALL

;*************************************************
;***** Camera
;*************************************************
;device_name
;device_id
;target_dir

;*************************************************
;***** Source
;*************************************************
  v4l2_device /dev/video0
;netcam_url

;*************************************************
;***** Image
;*************************************************
width 640
height 480
framerate 15

;*************************************************
;***** Overlays
;*************************************************
text_right %Y-%m-%d\n%T

;*************************************************
;***** Method
;*************************************************
emulate_motion off
threshold 1500

;*************************************************
;***** Masks
;*************************************************
noise_level 32
despeckle_filter EedDl

;*************************************************
;***** Detection
;*************************************************
minimum_motion_frames 1
event_gap 60
pre_capture 3
post_capture 10

;*************************************************
;***** Scripts
;*************************************************
on_event_start
on_event_end

;*************************************************
;***** Picture
;*************************************************
picture_output off
picture_filename %v-%Y%m%d%H%M%S-%q

;*************************************************
;***** Movie
;*************************************************
movie_output on
movie_max_time 120
movie_quality 45
movie_container mkv
movie_filename %v-%Y%m%d%H%M%S

;*************************************************
;***** Web Control
;*************************************************
webcontrol_port 8080
webcontrol_localhost off
webcontrol_parms 2

;*************************************************
;***** Web Stream
;*************************************************
stream_preview_scale 25
stream_preview_method mjpg

;*************************************************
; Device config files - One for each device.
;*************************************************
 camera /etc/motionplus/camera1-dist.conf
; camera /usr/etc/motionplus/camera2.conf
; camera /usr/etc/motionplus/camera3.conf
; sound /usr/etc/motionplus/sound1.conf

;*************************************************
; Directory to read '.conf' files for devices.
;*************************************************
; config_dir /usr/etc/motionplus/conf.d

### **Here is my camera1-dist.conf file:**

; This config file was generated by motionplus 0.2.1

;*************************************************
; Configuration options specific to camera 1
;*************************************************

;*************************************************
;***** Camera
;*************************************************
device_name MyCam1
device_id 101

;*************************************************
;***** Source
;*************************************************
;netcam_url rtsp://yourcamera1ip:port/camera/specific/url
 libcam_device camera0

;*************************************************
;***** Image
;*************************************************
width 640
height 480

;*************************************************
;***** Overlays
;*************************************************
text_left CAMERA 1

;*************************************************
;***** Movie
;*************************************************
movie_filename CAM01_%t-%v-%Y%m%d%H%M%S`

I'm assuming I need to fix something in the main config files and the camera config files but I am not quite sure what. I would appreciate any help.

**Comment by sroller** [comment]
What is the problem you want to solve? The screenshot doesn't show anything.
You should configure the logfile and post it here next time. Happy to help.

**Comment by Mr-Dave** [comment]
The message `unable to start MHD` almost always means that there is a different instance of Motion or Motionplus is running and using the port.
